docs/en/changes/changes.md

```diff
@@ -130,6 +130,7 @@
 * Add logo for HTTPX.
 * Refactor: optimize the attached events visualization in the trace widget.
 * Update BanyanDB client to 0.3.0.
+* Add AWS DynamoDB menu.
```

Comment:
More change logs when you sync submodule. For this time, from efed817f73fd227244e60efbb3c26b591165d47e to latest.